CMakeLists.txt

```diff
@@ -75,8 +75,8 @@ endif(BUILD_TESTS)
 set(Boost_USE_STATIC_LIBS ON)
 set(Boost_USE_DEBUG_RUNTIME OFF)
 #set_property(GLOBAL PROPERTY Boost_USE_MULTITHREADED ON)
-set(BOOST_PKG_VERSION "1.79.0" CACHE STRING "")
-find_package(Boost ${BOOST_PKG_VERSION} REQUIRED COMPONENTS system log log_setup thread program_options date_time filesystem)
```

Comment:
you can use EXACT to enforce a specific version (default behavior is greater than or equal to specified version) : `find_package(Boost ${BOOST_PKG_VERSION} EXACT REQUIRED COMPONENTS`

Also, maybe add asio and http to the list of COMPONENTS, or do that later in another PR.

Reply:
It turns out asio and the other components we discussed with Marco are header only and can't be included here or it breaks.

Do we want EXACT here? Ideally we'd be as backwards & forwards compatible as possible.

Reply:
Previously we did not enforce an exact version, so when people tried building the local proxy with boost 1.81 while we were targeting 1.79, they got a build error instead of a cmake validation error